src/server/config/schema.js

```diff
@@ -52,6 +52,7 @@ module.exports = () => Joi.object({
       disableProtection: Joi.boolean().default(false),
       token: Joi.string().optional().notes('Deprecated')
     }).default(),
+    staticDirectoryListings: Joi.boolean().default(Joi.ref('$dev')),
```

Comment:
Yes, it defaults to the value of `$dev`, which will only be true when in development mode.